content/en/agent/basic_agent_usage/_index.md

```diff
@@ -22,152 +22,60 @@ further_reading:
-
-Two optional processes are spawned by the Agent if enabled in the `datadog.yaml` configuration file:
-
-* The APM Agent is a process to collect [traces][2] (enabled by default).
-* The Process Agent is a process to collect live process information. By default, it only collects available containers, otherwise it is disabled.
-
-On Windows the services are listed as:
-
-| Service               | Description           |
-|-----------------------|-----------------------|
-| DatadogAgent          | Datadog Agent         |
-| datadog-trace-agent   | Datadog Trace Agent   |
-| datadog-process-agent | Datadog Process Agent |
-
-By default the Agent binds 3 [ports][3] on Linux and 4 on Windows and OSX:
+You can manage your Agent installation using the Datadog Agent Manager GUI or from the command line.
 
-| Port | Description                                                                                 |
-|------|---------------------------------------------------------------------------------------------|
-| 5000 | Exposes runtime metrics about the Agent.                                                    |
-| 5001 | Used by the Agent CLI and GUI to send commands and pull information from the running Agent. |
-| 5002 | Serves the GUI server on Windows and OSX.                                                   |
-| 8125 | Used for the DogStatsD server to receive external metrics.                                  |
+### Datadog Agent Manager GUI
 
-For information on configuring the ports, see [Network Traffic][4].
+<div class="alert alert-info">The Agent GUI is not supported on 32-bit Windows platforms.</div>
 
-### Collector
+Use the Datadog Agent Manager GUI to:
+- View the status information for your Agent
+- View all running checks
+- View the Agent log
+- Edit the Agent configuration file (`datadog.yaml`)
+- Add or edit Agent checks
+- Send flares
 
-The collector gathers all standard metrics every 15 seconds. Agent v6 embeds a Python 2.7 interpreter to run integrations and [custom checks][5].
+The Datadog Agent Manager GUI is enabled by default on Windows and macOS, and runs on port `5052`. Use the `datadog-agent launch-gui` command to open the GUI in your default web browser.
 
-### Forwarder
+You can change the GUIs default port in your `datadog.yaml` configuration file. To disable the GUI, set the port's value to `-1`. The GUI is disabled by default on Linux.
 
-The Agent forwarder send metrics over HTTPS to Datadog. Buffering prevents network splits from affecting metric reporting. Metrics are buffered in memory until a limit in size or number of outstanding send requests are reached. Afterwards, the oldest metrics are discarded to keep the forwarder's memory footprint manageable. Logs are sent over an SSL-encrypted TCP connection to Datadog.
+GUI requirements:
+- Cookies must be enabled in your browser. The GUI generates and saves a token in your browser which is used for authenticating all communications with the GUI server.
```

Comment:
```suggestion
- Cookies must be enabled in your browser. The GUI generates and saves a token in your browser, which is used for authenticating all communications with the GUI server.
```

Or 'that' instead of 'which', no comma